test/quantization/test_quantize_jit.py

```diff
@@ -22,17 +22,19 @@
     default_histogram_observer,
     default_eval_fn,
     fuse_modules,
+    quantize_jit,
+    quantize_dynamic_jit,
 )
 
-# torch.quantization.quantize_script
-from torch.quantization.quantize_script import script_qconfig
-from torch.quantization.quantize_script import prepare_script
-from torch.quantization.quantize_script import convert_script
-from torch.quantization.quantize_script import quantize_script
-from torch.quantization.quantize_script import prepare_dynamic_script
-from torch.quantization.quantize_script import convert_dynamic_script
-from torch.quantization.quantize_script import quantize_dynamic_script
-from torch.quantization.quantize_script import fuse_conv_bn_script
+# torch.quantization.quantize_jit
+from torch.quantization.quantize_jit import (
+    script_qconfig,
+    prepare_jit,
+    convert_jit,
+    prepare_dynamic_jit,
+    convert_dynamic_jit,
+    fuse_conv_bn_jit,
```

Comment:
nit: Can you put them in alphabetical order -- easier to find. Also, in my personal opinion having separate lines for import (instead of `()`) looks better :)

Reply:
Sure, will put them in alphabetical order, `()` has less things to type I think?